references/detection/train_pytorch.py

```diff
@@ -179,7 +179,11 @@ def main(args):
     val_set = DetectionDataset(
         img_folder=os.path.join(args.val_path, 'images'),
         label_path=os.path.join(args.val_path, 'labels.json'),
-        img_transforms=T.Resize((args.input_size, args.input_size)),
+        img_transforms=T.Resize((args.input_size, args.input_size)) if not args.rotation else None,
+        sample_transforms=T.SampleCompose([
+            T.RandomRotate(0, expand=True),
```

Comment:
Just to clarify: here we do this convert the boxes to polygons, correct?
If so, shouldn't we remove this and simply pass `use_polygons=True` to the dataset constructor?

Reply:
that is True